components/ListElement/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@gemeente-denhaag/listelement",
```

Comment:
"listelement" shouldn't this just be "list"? Currently you're suggestion other elements are no elements xD

@bddjong any thoughts?

Reply:
'list' already exists

Reply:
I would suggest following the utrecht approach. Separating the unordered list and ordered list into separate components, and keeping the naming as close to the element names as possible.

`denhaag/unordered-list` and `denhaag/ordered-list`

https://github.com/nl-design-system/utrecht/tree/main/components/unordered-list
https://github.com/nl-design-system/utrecht/tree/main/components/ordered-list

This will also allow us to use `@extend .utrecht-unordered-list` for example, to use their CSS and design tokens.